### ISSUE TITLE
Won't boot to USB Drive

### ISSUE THREAD

**Issue by safetychief8** [pathartl/BleemSync#51]
My USB drive won't boot to the hacked items.

I'm posting the details from the regional file in the system folder, maybe someone understands what's going on. I'm using the Sandisk Cruzer USB 2.0 drive. Below are some of the details in the reigional file:

 "DISC_ID" INTEGER NOT NULL CONSTRAINT "PK_DISC" PRIMARY KEY AUTOINCREMENT,
    "GAME_ID" INTEGER NOT NULL,
    "DISC_NUMBER" INTEGER NOT NULL,
    "BASENAME" TEXT NULL,
    CONSTRAINT "FK_DISC_GAME_GAME_ID" FOREIGN KEY ("GAME_ID") REFERENCES "GAME" ("GAME_ID") ON DELETE CASCADE
)[//‚etableLANGUAGE_SPECIFICLANGUAGE_SPECIFICCREATE TABLE "LANGUAGE_SPECIFIC" (
    "LANGUAGE_ID" TEXT NOT NULL CONSTRAINT "PK_LANGUAGE_SPECIFIC" PRIMARY KEY,
    "DEFAULT_VALUE" TEXT NULL,
    "VALUE" TEXT NULL
)AU/ indexsqlite_autoindex_LANGUAGE_SPECIFIC_1LANGUAGE_SPECIFIC       P++Ytablesqlite_sequencesqlite_sequenceCREATE TABLE sqlite_sequence(name,seq)‚m…=tableGAMEGAMECREATE TABLE "GAME" (
    "GAME_ID" INTEGER NOT NULL CONSTRAINT "PK_GAME" PRIMARY KEY AUTOINCREMENT,
    "GAME_TITLE_STRING" TEXT NULL,
    "PUBLISHER_NAME" TEXT NULL,
    "RELEASE_YEAR" INTEGER NOT NULL,
    "PLAYERS" INTEGER NOT NULL,
    "RATING_IMAGE" TEXT NULL,
    "GAME_MANUAL_QR_IMAGE" TEXT NULL,
    "LINK_GAME_ID" TEXT NULL

**Comment by safetychief8** [comment]
...I also have loaded 37 games loaded onto the USB drive, don't know if that will also cause issues.

**Comment by acidrain97** [comment]
I had the same problem. I was using San disk 16 or 32 gb would not work. I used some off brand USB's 2 gig and 4 gig everything worked. What is so special about the size or brand I have no idea.

**Comment by rick2890** [comment]
SanDisk SDCZ50-016G-B35 and delete the bleemsync folder once you have run bleemsync.exe

**Comment by acidrain97** [comment]
> SanDisk SDCZ50-016G-B35 and delete the bleemsync folder once you have run bleemsync.exe

did that no go.

I'm using a SanDisk cruzer glider 16gb sdcz60-016g

**Comment by safetychief8** [comment]
I actually used a SanDisk micro SD card with a USB adapter, that worked ok, just a file issue I have to back track and check the NFL gameday and NCAA game breaker files, that's when it would flash but wouldn't boot. It did before I added those 2 roms. I tried the SanDisk SDCZ36-128G-B35, but the light in the console wouldn't flash. I'll try to repair the files then see if I can get the SanDisk USB drive to work, if not, I'll stick with the Micro SD with the adapter.
